src/SFML/Graphics/Shader.cpp

```diff
@@ -989,12 +989,6 @@ Shader::CurrentTextureType Shader::CurrentTexture;
 Shader::Shader() = default;
 
 
-////////////////////////////////////////////////////////////
-Shader::~Shader()
-{
-}
```

Comment:
What about the declaration in the header file?

Reply:
I'm not actually sure how CI passes with this removed. I suppose it's something related to how you don't need to define symbols that don't get used and the configurations that use this half of the `Shader` implementation never actually try to use a `Shader`. I think this needs to be restored.